turtlebot_follower/launch/follower.launch

```diff
@@ -40,6 +40,43 @@
     </node>
   </group>
   <group if="$(arg simulation)">
-    <!-- not supported.. -->
+    <node pkg="nodelet" type="nodelet" name="follower_velocity_smoother"
```

Comment:
This content looks the same to me as the include file above except for the nodelet manager. If we launch a manager as described in my comment below. We could add an argument to the included launch file which passes the manager name, and then in the regular block pass `/mobile_base_nodelet_manager` but in the simulation case pass `/camera/camera_nodelet_manager`

How to pass an argument to in include tag: http://wiki.ros.org/roslaunch/XML/include
How to use an argument in a launch file: http://wiki.ros.org/roslaunch/XML/arg